### PR TITLE
New version: Chromium.ChromeDriver version 123.0.6312.58

### PR DESCRIPTION
**Pull request by schembriaiden**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146062)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-03-22 14:02:02.296 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Chromium.ChromeDriver.123.0.6312.58\extracted\chromedriver.exe
> 2024-03-22 14:01:12.608 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Chromium.ChromeDriver.123.0.6312.58\extracted\chromedriver.exe

(Automated response - build 872.)

**Comment by vikingnope** [comment]
Seems like the chromedriver.exe is not an installer anymore, it is just a file that needs to be run:

image

image


Don't know if that affects it

**Review by stephengillie — DISMISSED** [review]

File: manifests/c/Chromium/ChromeDriver/123.0.6312.58/Chromium.ChromeDriver.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created with komac v2.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Chromium.ChromeDriver
+PackageVersion: 123.0.6312.58
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: chromedriver.exe
```
Comment (stephengillie) [inline manifests/c/Chromium/ChromeDriver/123.0.6312.58/Chromium.ChromeDriver.installer.yaml:9]:
```suggestion
- RelativeFilePath: chromedriver-Win64/chromedriver.exe
```
The driver appears to have moved to within the `Win64` folder - does this imply that the `x86` version has been depreciated?

**Comment by vikingnope** [comment]
since there is a win64 and win32 file, what should the relative file path be?

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-03-25 13:02:45.798 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Chromium.ChromeDriver.123.0.6312.58\extracted\chromedriver-win64/chromedriver.exe

(Automated response - build 873.)

**Comment by stephengillie** [comment]
Is the `x86` version having difficulty? This package installs and launches normally on a 64-bit Windows 10 VM:

image

**Comment by vikingnope** [comment]
I have not tried it on a 32-bit system, and I cannot try it because my installs are 64 bit

**Comment by stephengillie** [comment]
Is it possible that this version is 64-bit only? Should the `x86` installer data be removed?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**